interfaces/cython/cantera/reactor.pyx

```diff
@@ -1071,6 +1133,48 @@ cdef class ReactorNet:
         self.net.getState(&y[0])
         return y
 
+    def get_derivative(self, k):
+        """
+        Get the k-th time derivative of the state vector of the reactor network.
+        """
+        if not self.n_vars:
+            raise CanteraError('ReactorNet empty or not initialized.')
+        cdef np.ndarray[np.double_t, ndim = 1] dky = np.zeros(self.n_vars)
+        self.net.getDerivative(k, & dky[0])
+        return dky
+
+    property last_order:
+        """
+        get the integration order of the last step evaluation
+        """
+        def __get__(self):
+            return self.net.lastOrder()
+
+    property advance_limits:
+        """
+        Get or set absolute limits for state changes during `ReactorNet.advance`
+        (positive values are considered; negative values are ignored). 
+        """
+        def __get__(self):
+            cdef np.ndarray[np.double_t, ndim=1] limits = np.empty(self.n_vars)
+            self.net.getAdvanceLimits(&limits[0])
+            return limits
+
+        def __set__(self, limits):
+            if len(limits) != self.n_vars:
+                raise ValueError('array must be of length n_vars')
+
+            cdef np.ndarray[np.double_t, ndim=1] data = \
+                np.ascontiguousarray(limits, dtype=np.double)
+            self.net.setAdvanceLimits(&data[0])
+
+    def set_advance_limit(self, int index, double limit):
```

Comment:
could be removed

Reply:
I think this is the preferred name instead of the property.

Reply:
Eliminated, as redundant (now handled by reactor object).